src/isp_trace_parser/get_data.py

```diff
@@ -1,12 +1,739 @@
+        ...     reference_year_mapping={2024: 2011, 2025: 2012},
+        ...     project=["Bango 973 Wind Farm", "Bodangora Wind Farm"],
+        ...     directory="parsed_data/project"
+        ... ) # doctest: +SKIP
+                         datetime     value              project
+        0     2023-07-01 00:30:00  0.244017  Bodangora Wind Farm
+        1     2023-07-01 00:30:00  0.436084  Bango 973 Wind Farm
+        2     2023-07-01 01:00:00  0.273899  Bodangora Wind Farm
+        3     2023-07-01 01:00:00  0.489390  Bango 973 Wind Farm
+        4     2023-07-01 01:30:00  0.340990  Bodangora Wind Farm
+        ...                   ...       ...                  ...
+        70171 2025-06-30 23:00:00  0.065651  Bango 973 Wind Farm
+        70172 2025-06-30 23:30:00  0.182044  Bodangora Wind Farm
+        70173 2025-06-30 23:30:00  0.049964  Bango 973 Wind Farm
+        70174 2025-07-01 00:00:00  0.218949  Bodangora Wind Farm
+        70175 2025-07-01 00:00:00  0.037577  Bango 973 Wind Farm
+        <BLANKLINE>
+        [70176 rows x 3 columns]
+
+        Include reference_year column to identify which reference year was used:
+
+        >>> get_project_multiple_reference_years(
+        ...     reference_year_mapping={2024: 2011, 2025: 2012},
+        ...     project=["Bango 973 Wind Farm", "Bodangora Wind Farm"],
+        ...     directory="parsed_data/project",
+        ...     select_columns=["datetime", "value", "project", "reference_year"]
+        ... ) # doctest: +SKIP
+                         datetime     value              project  reference_year
+        0     2023-07-01 00:30:00  0.244017  Bodangora Wind Farm            2011
+        1     2023-07-01 00:30:00  0.436084  Bango 973 Wind Farm            2011
+        2     2023-07-01 01:00:00  0.273899  Bodangora Wind Farm            2011
+        3     2023-07-01 01:00:00  0.489390  Bango 973 Wind Farm            2011
+        4     2023-07-01 01:30:00  0.340990  Bodangora Wind Farm            2011
+        ...                   ...       ...                  ...             ...
+        70171 2025-06-30 23:00:00  0.065651  Bango 973 Wind Farm            2012
+        70172 2025-06-30 23:30:00  0.182044  Bodangora Wind Farm            2012
+        70173 2025-06-30 23:30:00  0.049964  Bango 973 Wind Farm            2012
+        70174 2025-07-01 00:00:00  0.218949  Bodangora Wind Farm            2012
+        70175 2025-07-01 00:00:00  0.037577  Bango 973 Wind Farm            2012
+        <BLANKLINE>
+        [70176 rows x 4 columns]
+    """
+    return _query_parquet_multiple_reference_years(
+        reference_year_mapping=reference_year_mapping,
+        directory=directory,
+        filters={"project": project},
+        year_type=year_type,
+        select_columns=select_columns,
+    )
+
+
+@validate_call
+def get_zone_multiple_reference_years(
+    reference_year_mapping: dict[int, int],
+    zone: str | List,
+    resource_type: str | Path,
```

Comment:
Need this change otherwise pydantic will raise an error when passing in a list

```suggestion
    resource_type: str | List,
```